libs/velo-external-db-types/src/index.ts

```diff
@@ -91,7 +91,7 @@ export type AdapterAggregation = {
 export interface IDataProvider {
```

Comment:
IMHO, a genaric pool type should be here, so the `IDataProvider` should look like this: `IDataProvider<T>`
and then add to the this iterface the must have variables for example pool - each DataProvider must have a pool variable and the type of this variable will be T (the pool type)